dev-docs/source/Testing/MuonAnalysis_test_guides/Muon_ARGUS.rst

```diff
@@ -75,7 +75,6 @@ This users the same sample as the single pulse test
 	- Click **Guess Alpha**, should get ``0.91``
 - Load the next run
 - Go to the **Fitting** tab
-	- Add a **StaticKuboToyabeTimeExpDecay** and **FlatBackground**
 	- Click the **Fit** button
```

Comment:
Just to clarify - is this manual test one to test that you still get a result even if you don't add anything?

Reply:
you will still have it from line 61